### PR TITLE
chore: improve omnichannel tests reliability

### PR DESCRIPTION
**Pull request by aleksandernsilva**

### CODE REVIEW

File: apps/meteor/tests/e2e/page-objects/fragments/home-sidenav.ts [hunk 1]
```diff
@@ -64,10 +64,14 @@ export class HomeSidenav {
...
-		await this.page.locator('role=search >> role=searchbox').type(name);
-		await this.page.locator(`role=search >> role=listbox >> role=link >> text="${name}"`).click();
+	async openChat(name: string, search = true): Promise<void> {
+		if (search) {
+			await this.page.locator('role=navigation >> role=button[name=Search]').click();
+			await this.page.locator('role=search >> role=searchbox').type(name);
+			await this.page.locator(`role=search >> role=listbox >> role=link >> text="${name}"`).click();
+		} else {
+			await this.getSidebarItemByName(name).click();
+		}
```
Comment (ggazzo) [inline apps/meteor/tests/e2e/page-objects/fragments/home-sidenav.ts:74]: why not call `getSidebarItemByName`directly ?

**Review by ggazzo — CHANGES_REQUESTED** [review]
just keep in mind  that you are actually changing the test before people were opening through search and now the test is being performed agains the regular sidebar


btw is this a problem yet? I thought it was not happening anymore those kind of errors

File: apps/meteor/tests/e2e/page-objects/fragments/home-sidenav.ts (hunk 1 above)
Comment (aleksandernsilva) [reply apps/meteor/tests/e2e/page-objects/fragments/home-sidenav.ts]: > just keep in mind that you are actually changing the test before people were opening through search and now the test is being performed agains the regular sidebar

That's the objective ;) I want **Omnichannel tests** to get the room directly from the sidebar. This should prevent a problem we are facing where it takes some time for the room to show up in the queue. I created this draft specifically to confirm that.

> why not call `getSidebarItemByName` directly ?

Actually might be a good idea.

**Review by MartinSchoeler — APPROVED** [review]

**Review by MartinSchoeler — DISMISSED** [review]

File: apps/meteor/tests/e2e/page-objects/omnichannel-triggers.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export class OmnichannelTriggers {
 	}
 
 	get btnNew(): Locator {
-		return this.page.locator('text=New');
+		return this.page.locator('button >> text=New');
```
Comment (ggazzo) [inline apps/meteor/tests/e2e/page-objects/omnichannel-triggers.ts:16]:
```suggestion
		return this.page.locator('role=button[name="New"');
```

**Review by ggazzo — APPROVED** [review]

**Pull request merged**